lib/clowder-common-ruby/version.rb

```diff
@@ -1,3 +1,3 @@
 module ClowderCommonRuby
-  VERSION = '0.5.1'.freeze # Patch version is being automatically bumped
+  VERSION = '0.5.2'.freeze # Patch version is being automatically bumped
```

Comment:
I don't think we need a version bump, maybe not even the types.rb change as the next build tomorrow would to these for us.